developer/core/desktop/16.0/c-api/state-api.php

```diff
@@ -491,3 +491,84 @@ enum.
     </dl>
   </section>
 </article>
+
+
+
+<article id="km_kbp_process_event">
```

Comment:
```suggestion
<article id="km_kbp_event">
```

km_kbp_process_event is already used at l.437